docker/docker-commands

```diff
@@ -0,0 +1,11 @@
+-- запуск базы --
```

Comment:
English please. Also this file should be called `README.md` so that it would show up in Github

Reply:
Fill fix it too.